core/controllers/feedback_test.py

```diff
@@ -336,3 +339,109 @@ def _get_email(n):
             self.assertEqual(
                 response_dict['messages'][num + 1]['text'],
                 'New Message %s' % num)
+
+    def _return_true(self, thread_id, exploration_id):
+        return True
+
+    def _return_null(self, param_one, param_two, param_three, param_four):
+        return None
+
+    def test_create_fetch_and_action_suggestions(self):
+        # Create suggestions.
+        self.login(self.EDITOR_EMAIL)
+        response = self.testapp.get('/create/%s' % self.EXP_ID)
+        csrf_token = self.get_csrf_token_from_response(response)
+        # Create suggestions.
+        self.post_json(
```

Comment:
I had used the same suggestion details for all suggestions because it doesn't matter too much, have changed them now though.
The field named old_content is actually the new content, have changed the name.